src/Lecture2.hs

```diff
@@ -97,7 +117,8 @@ spaces.
 
 🕯 HINT: look into Data.Char and Prelude modules for functions you may use.
 -}
-dropSpaces = error "TODO"
+dropSpaces :: String -> String
+dropSpaces = (takeWhile (/= ' ') . dropWhile(== ' '))
```

Comment:
That's a correct solution! 👏🏻 
You can also use the [isSpace](https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-Char.html#v:isSpace) function from the `Data.Char` module if you import it.

Reply:
Yeah, I've found "isSpace" and I wasn't sure about how to import Data.Char correctly.